app/app.pro

```diff
@@ -466,3 +466,6 @@ contains(DEFINES,FETCH_LOCAL_METADATA) {
 }
 
 INSTALLS += target
+
+OTHER_FILES += \
```

Comment:
This is probably auto generated and not needed since we include by wildcard above.